README.md

```diff
@@ -1,20 +1,51 @@
 # cyclic-voltammetry
-dependencies
 
-```
+Streamlit App to explore the data in the [echemdb repository](https://github.com/echemdb/electrochemistry-data), containing cyclic voltammograms (CVs) for single crystal electrodes.
+
+A static version of the database can be found on [echemdb.org/cv](https://www.echemdb.org/cv/).
+
+## Installation
+
+### Install with pip
+
+```sh
 pip install -r requirements.txt
 ```
 
-db
+Continue below (Download the database).
+
+### Developers
 
+Clone the repository.
+
+```sh
+git clone git@github.com:DunklesArchipel/cyclic-voltammetry.git
+cd cyclic-voltammetry
 ```
+
+Install dependencies with conda (or mamba).
+
+```sh
+mamba env create -f environment.yaml
+mamba activate echemdb_app
+```
+
+### Download the database
+
+Download the recent version of the database with the [unitpackage](https://echemdb.github.io/unitpackage/) API. Rerun the following lines, when the database changes.
+
+```sh
+python
 from unitpackage.collection import Collection
-db = Collection.from_remote()
+db = Collection.from_remote() # the download may take a few minutes
 db.save_entries('entries')
+exit()
```

Comment:
We also have the file `db/_persist_db_local.py`, which could be executed instead. Haven't maintained the README yet 😓 

Running `python db/_persist_db_local.py` would accomplish the same thing

Reply:
Sure, an approach to download the database without moving into Python is possibly a good idea.

Feel free to include it in this PR or fix it later, when you work more on the documentation.